### ISSUE TITLE
Merging styles across libraries

### ISSUE THREAD

**Issue by KacperBieda** [oodesign/merge-duplicate-symbols#62]
Hey,

I bought this plugin and I'm wondering if there is any way to merge styles across multiple libraries. I was planning to use to make that happen "list merge", but from what I see I have to select manually all hundreds of styles. 😅

Short story - I've splitted one big library to 40ty smaller library files. The downside of this move was an issue with the styles - now each file has it own styles which are the same. I'm looking for a way to connect all the styles from other libraries to a single library. 

![image](https://user-images.githubusercontent.com/35021794/64280279-0da92400-cf51-11e9-8312-3b79a0eb2464.png)

Basically I want all these libraries to download the styles from $basic-styles library, and  remove all duplicated styles from $basic in other libraries.

**Comment by oodesign** [comment]
Hi @KacperBieda!

Thanks for getting the plugin! Means a lot to us!

Regarding your question, I'd say it's a pretty common case, and I think Merge Duplicates may be useful in there. The only 'ugly' part of it is that you'll have to do it for all of your "child" libraries (40??? Makes sense as I guess you're splitting by component, but... wow!)  😅.

My best guess would be to:

1. Open one of the "child" libraries
2. Make sure that, in the Sketch Preferences (or linked libraries if you use Abstract), there's only the "parent" library (the one with the right styles) linked (and not the child ones).
3. Run Merge Duplicate > Text Styles > **Merge styles with the same name**
4. Go one by one selecting the library style instead of the local style
5.  And that's it... all of your local styles will be replaced (also overrides) for the library ones
6. And... do this same stuff for the rest of your 40 child files 🤦‍♂️🤷‍♂️

Would that help?
